src/zope/security/checker.py

```diff
@@ -24,10 +24,10 @@
 (:func:`getCheckerForInstancesOf`, :func:`selectChecker`).
 
 This module is accelerated with a C implementation on CPython by
-default. If the environment variable ``PURE_PYTHON`` is set (to any
-value) before this module is imported, the C extensions will be
-bypassed and the reference Python implementations will be used. This
-can be helpful for debugging and tracing.
+default. If the environment variable ``PURE_PYTHON`` is set to ``1``
```

Comment:
I am not yet fully happy (the text still does not yet tell that the envvar is expected to have an `int` value) but the text is not important enough to prevent a merge.

Reply:
I don't like to leave it like that, but I could not find better words